### PR TITLE
add ids to categories array,

### PR DESCRIPTION
**Pull request by chrispetito**
Co-authored-by: James <15709701+Shadowborn@users.noreply.github.com>
Co-authored-by: Josh <44819171+Joshua-Disney@users.noreply.github.com>
Co-authored-by: Kayla <44589691+blackmacaroon@users.noreply.github.com>
Co-authored-by: Victor <46913219+victor082@users.noreply.github.com>

# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/ProjectCard/Project.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const Project = props => {
   const { project } = props
   const { display_image, name, description, hosted_url, category, is_approved } = project
 
+  console.log(category)
```
Comment (lappjeff) [inline src/components/ProjectCard/Project.js]: Only YOU can prevent random console logs 🚭

File: src/components/SubmissionForm/ProjectForm.js [hunk 2]
```diff
@@ -45,11 +46,20 @@ const ProjectForm = props => {
     let submitted_at = moment().format("MMMM Do YYYY, h:mm:ss a")
     let display_image = props.display_image && props.display_image
     let newPost = {
-      ...state,
+      //send all of state to BE except category_name and error_message (will return a 500 if you include these)
```
Comment (lappjeff) [inline src/components/SubmissionForm/ProjectForm.js]: Feel free to remove the category_name property from state and props if we no longer have a use case for it 👍

File: src/components/SubmissionForm/ProjectForm.js [hunk 3]
```diff
@@ -45,11 +46,20 @@ const ProjectForm = props => {
...
+      backend_url: state.backend_url,
+      name: state.name,
+      category_id: state.category_id.id,
+      description: state.description,
+      // error_message: state.error_message,
       submitted_at,
       display_image,
     }
 
+    console.log(newPost)
```
Comment (lappjeff) [inline src/components/SubmissionForm/ProjectForm.js]: Kill it

File: src/components/SubmissionForm/ProjectForm.js (hunk 2 above)
Comment (chrispetito) [reply src/components/SubmissionForm/ProjectForm.js]: Removed!

File: src/components/SubmissionForm/ProjectForm.js (hunk 3 above)
Comment (chrispetito) [reply src/components/SubmissionForm/ProjectForm.js]: Killed!

File: src/components/ProjectCard/Project.js (hunk 1 above)
Comment (chrispetito) [reply src/components/ProjectCard/Project.js]: Removed!

**Pull request merged**